Sources/Amplitude/Plugins/iOS/UIKitUserInteractions.swift

```diff
@@ -60,8 +107,26 @@ class UIKitUserInteractions {
             swizzledImp,
             method_getTypeEncoding(swizzledMethod))
     }
+
+    private static let accessibilityHierarchyParser = UIKitAccessibilityHierarchyParser()
+
+    private static func findTargetUnderTap(for gestureRecognizer: UIGestureRecognizer) -> AccessibilityTarget? {
+        guard let view = gestureRecognizer.view else { return nil }
+
+        let tapLocation = gestureRecognizer.location(in: nil)
+
+        guard let target = accessibilityHierarchyParser.parseAccessibilityElement(on: tapLocation, in: view) else { return nil }
+
+        if let targetView = target.object as? UIView, let control = targetView.controlInHierarchy, !control.allControlEvents.isEmpty {
+            return nil
+        }
+
+        return target
+    }
 }
 
+private class _AmplitudeSwiftUIGestureRecognizer: UITapGestureRecognizer {}
```

Comment:
I've added this to be able to later distinguish the custom gesture recognizer from the others in the window.